e-commerce-project/src/app/app.module.ts

```diff
@@ -1,18 +1,19 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
+import { FormsModule }   from '@angular/forms';
 import { AppRoutingModule } from './app-routing.module';
```

Comment:
Below I wrapped <input> tags with <form> tag.

When we have <form> on our page and when user submits the form, browser (default HTML 5 behavior) will append at the end of url some query with names of input fields ("http://localhost:4200/?username=guest&pass=123456") and browser will navigate to the page (browser will treat it as a new page).
Please try / test that behavior.

In order to prevent that behavior we will import angular form module 'FormsModule' into 'imports: [..]' array in app.module file.